src/components/media-player/ha-media-player-browse.ts

```diff
@@ -92,11 +94,54 @@ export class HaMediaPlayerBrowse extends LitElement {
     this._navigate(item);
   }
 
+  private _renderError(err: { message: string; code: string }) {
+    if (err.message === "Path does not exist.") {
```

Comment:
After a backend change this should be:
```suggestion
    if (err.code === "no_media_folder") {
```